sdk/eventhub/Azure.ResourceManager.EventHubs/tests/Tests/ScenarioTests.EventHubsTests.CRUDAuthorizationRules.cs

```diff
@@ -99,40 +100,40 @@ public async Task EventhubCreateGetUpdateDeleteAuthorizationRules()
+            var updateEventhubAuthorizationRuleResponse =await EventHubsOperations.CreateOrUpdateAuthorizationRuleAsync(resourceGroupName,
                 namespaceName, eventHubName, authorizationRuleName, updateEventhubAuthorizationRuleParameter);
             Assert.NotNull(updateEventhubAuthorizationRuleResponse);
             Assert.AreEqual(authorizationRuleName, updateEventhubAuthorizationRuleResponse.Value.Name);
             Assert.True(updateEventhubAuthorizationRuleResponse.Value.Rights.Count == updateEventhubAuthorizationRuleParameter.Rights.Count);
             Assert.True(isContains(updateEventhubAuthorizationRuleParameter.Rights, updateEventhubAuthorizationRuleResponse.Value.Rights));
             // Get the updated Eventhub AuthorizationRule
-            var getEventhubAuthorizationRuleResponse = await EventHubsOperations.GetAuthorizationRuleAsync(resourceGroup, namespaceName, eventHubName,
+            var getEventhubAuthorizationRuleResponse = await EventHubsOperations.GetAuthorizationRuleAsync(resourceGroupName, namespaceName, eventHubName,
                 authorizationRuleName);
             Assert.NotNull(getEventhubAuthorizationRuleResponse);
             Assert.AreEqual(authorizationRuleName, getEventhubAuthorizationRuleResponse.Value.Name);
             Assert.True(getEventhubAuthorizationRuleResponse.Value.Rights.Count == updateEventhubAuthorizationRuleParameter.Rights.Count);
             Assert.True(isContains(updateEventhubAuthorizationRuleParameter.Rights, getEventhubAuthorizationRuleResponse.Value.Rights));
             // Get the connectionString to the Eventhub for a Authorization rule created
-            var listKeysResponse = await EventHubsOperations.ListKeysAsync(resourceGroup, namespaceName, eventHubName, authorizationRuleName);
+            var listKeysResponse = await EventHubsOperations.ListKeysAsync(resourceGroupName, namespaceName, eventHubName, authorizationRuleName);
             Assert.NotNull(listKeysResponse);
             Assert.NotNull(listKeysResponse.Value.PrimaryConnectionString);
             Assert.NotNull(listKeysResponse.Value.SecondaryConnectionString);
             //New connection string
-            var regenerateConnection_primary = await EventHubsOperations.RegenerateKeysAsync(resourceGroup, namespaceName, eventHubName, authorizationRuleName, new RegenerateAccessKeyParameters(KeyType.PrimaryKey));
+            var regenerateConnection_primary = await EventHubsOperations.RegenerateKeysAsync(resourceGroupName, namespaceName, eventHubName, authorizationRuleName, new RegenerateAccessKeyParameters(KeyType.PrimaryKey));
             Assert.NotNull(regenerateConnection_primary);
-            Assert.AreNotEqual(listKeysResponse.Value.PrimaryConnectionString, regenerateConnection_primary.Value.PrimaryConnectionString);
+            // Assert.AreNotEqual(listKeysResponse.Value.PrimaryConnectionString, regenerateConnection_primary.Value.PrimaryConnectionString);
```

Comment:
I comment out this assertion and a few similar ones, due to the following error **in playback mode only**:
> Error Message:
>        Expected: not equal to "Sanitized"
>     But was:  "Sanitized"

Those could be due to the fact that test framework will sanitize some content it thinks sensitive during recording. So in the playback mode, the tests will fail.